Updated the clear-float technique to use flow-root (#21281)
mdn/content: files/en-us/web/css/clear/index.md

The floats that are relevant to be cleared are the earlier floats within the same [block formatting context](/en-US/docs/Web/Guide/CSS/Block_formatting_context).

> > **Note:** If an element contains only floated elements, its height collapses to nothing. If you want it to always be able to resize, so that it contains floating elements inside it, you need to change the display property to [flow-root](/en-US/docs/Web/CSS/display). 

Reviewer: ```suggestion
> **Note:** If an element contains only floated elements, its height collapses to nothing. If you want it to always be able to resize, so that it contains floating elements inside it, you need to change the display property to [flow-root](/en-US/docs/Web/CSS/display#flow-root). 
```

> > **Note:** If an element contains only floated elements, its height collapses to nothing. If you want it to always be able to resize, so that it contains floating elements inside it, you need to change the display property to [flow-root](/en-US/docs/Web/CSS/display#flow-root). 

Reviewer: ```suggestion
> **Note:** If an element contains only floated elements, its height collapses to nothing. If you want it to always be able to resize, so that it contains floating elements inside it, you need to change the [`display`](/en-US/docs/Web/CSS/display) property to [`flow-root`](/en-US/docs/Web/CSS/display#flow-root). 
```

> > **Note:** If an element contains only floated elements, its height collapses to nothing. If you want it to always be able to resize, so that it contains floating elements inside it, you need to change the display property to [flow-root](https://developer.mozilla.org/en-US/docs/Web/CSS/display). 

Reviewer: ```suggestion
> **Note:** If an element contains only floated elements, its height collapses to nothing. If you want it to always be able to resize, so that it contains floating elements inside it, you need to change the display property to [flow-root](/en-US/docs/Web/CSS/display). 
```

> > **Note:** If an element contains only floated elements, its height collapses to nothing. If you want it to always be able to resize, so that it contains floating elements inside it, you need to change the value of the [`display`](/en-US/docs/Web/CSS/display) property to [`flow-root`](/en-US/docs/Web/CSS/display#flow-root). 

Reviewer: ```suggestion
> **Note:** If an element contains only floated elements, its height collapses to nothing. If you want it to always be able to resize, so that it contains floating elements inside it, set the value of the element's [`display`](/en-US/docs/Web/CSS/display) property to [`flow-root`](/en-US/docs/Web/CSS/display#flow-root). 
```

>
> ```css
> #container {

> >   display:flow-root;

Reviewer: ```suggestion
>   display: flow-root;
```
